app/presenters/topic_presenter.rb

```diff
@@ -36,6 +36,7 @@ def links_payload
         linked_items: topic.guides.map(&:content_id),
         content_owners: content_owner_content_ids,
         organisations: [ServiceManualPublisher::GDS_ORGANISATION_CONTENT_ID],
+        parent: parents
```

Comment:
I still think doing this is our best option rather than tackling a custom link expansion.

We should probably add to the schema example: https://github.com/alphagov/govuk-content-schemas/blob/master/formats/service_manual_topic/frontend/examples/service_manual_topic.json